.github/workflows/feature-request-debugger.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: ./.github/actions/AddComment
         with:
           readonly: ${{ github.event.inputs.readonly }}
-          labels: feature request,debugger
+          labels: Feature Request,debugger
           ignoreLabels: "investigate: costing,investigate,external,by design,question,more info needed,language service,internal"
           createdAfter: "2024-07-22"
           addComment: "Thank you for your feature request. While we may not be able to implement it immediately, we will monitor community reactions to see how it fits into our backlog. Additionally, if you're working with GDB/LLDB, please note that the code is open source at https://github.com/microsoft/MIEngine/wiki/Contributing-Code . Your contributions are always welcome and appreciated."
```

Comment:
Can we also add the "help wanted" label as part of this? (and part of the bug bot)